SmartDeviceLink/SDLMenuManager.m

```diff
@@ -274,6 +407,20 @@ - (void)sdl_sendCurrentMenu:(SDLMenuUpdateCompletionHandler)completionHandler {
 
 #pragma mark - Helpers
 
+- (BOOL)checkUpdateMode:(SDLDynamicMenuUpdatesMode)dynamicMenuUpdatesMode {
+    switch (dynamicMenuUpdatesMode) {
+        case SDLDynamicMenuUpdatesModeForceOn:
+            return true;
+            break;
+        case SDLDynamicMenuUpdatesModeForceOff:
+            return false;
+            break;
+        case SDLDynamicMenuUpdatesModeOnWithCompatibility:
+            return [self.displayCapabilities.displayName isEqualToString:@"GEN3_8_INCH"] ? false : true;
```

Comment:
This is incorrect, it should be `return ![self.displayCapabilities.displayName isEqualToEnum:SDLDisplayTypeGen38Inch]`

Reply:
fixed